README.md

```diff
@@ -22,6 +22,33 @@ If you run into errors with `npm install` it may be due to node version, try `nv
 
 When running the `composer install/update`, Composer may prompt you for a GitHub OAuth token before it can fetch the `subscriptions` and `pre-orders` extensions from GitHub.
 
+### PHPStan workflow
+
+We are using [PHPStan](https://phpstan.org/) to perform static analysis of our code. Every PR will be analysed to check for issues via a GitHub workflow, which may include the following:
+ * Legitimate issues that you should resolve before merging, like missing null check or type checks.
+ * Error messages that are less clear if they need to be resolved, like array types not specifying a type for their contents.
+ * Error messages that indicate an error doesn't exist or could not be found.
+
+For the latter two situations, you will need to update the PHPStan baseline file, which you can do by running either of the following commands:
```

Comment:
It is not very clear to me when we need to update the baseline file. Is it after fixing the issues reported in the workflow?

Also, unrelated to this PR, but do you think it makes sense to add PHPStan to our pre-commit hooks?

Reply:
> To be honest, I am not 100% sure of the decision around what changes we should fix in the PR and which changes should drive us to update the baseline! :-/ (...)

Gotcha! I think we can improve this specific part, then, with some of the explanations you shared here

Reply:
It is good for me. Thanks for adding it. Approved 👍